util/symbols.py

```diff
@@ -22,14 +22,16 @@ def initialize(all_segments):
 
     all_symbols = []
     symbol_ranges = []
-
-    symbol_addrs_path = options.get_symbol_addrs_path()
+    func_addrs_lines = []
+    symbol_addrs_paths = options.get_symbol_addrs_paths()
 
     # Manual list of func name / addrs
-    if os.path.exists(symbol_addrs_path):
-        with open(symbol_addrs_path) as f:
-            func_addrs_lines = f.readlines()
+    for symbol_addrs_path in symbol_addrs_paths:
+        if os.path.exists(symbol_addrs_path):
+            with open(symbol_addrs_path) as f:
+                func_addrs_lines = func_addrs_lines + f.readlines()
 
+    if len(func_addrs_lines) > 0:
```

Comment:
1. `for x in y` is fine with an empty list, so you can remove this check and unindent everything.

2. `symbol_addrs_path.exists()` should work as they are `Path` types. means you can remove the `import os` as well

Reply:
symbol_addrs_path.exists() fails because the type is a string

Reply:
it should be a path...let's change that